qemu/tests/cfg/multi_nics_reboot.cfg

```diff
@@ -0,0 +1,10 @@
+- multi_nics_reboot:
+    virt_test_type = qemu libvirt
```

Comment:
If the `virt_test_type` is `qemu libvirt`, I think this file should be moved to `generic`, not `qemu`
Other LGTM.

Reply:
I have deleted it.thanks